quizer.py

```diff
@@ -28,6 +36,25 @@ def welcome_page():
         if not username:
             username = session['username'] = request.form['username']
         if username:
+            if not questions_list:
+                with open('data/quiz.csv', 'rb') as csvfile:
+                    spamreader = csv.reader(csvfile, delimiter=';', quotechar='|')
+                    for row in spamreader:
+                        q = unicode(row.pop(0), "utf8")
+                        correct_answer_letter = row.pop()
+                        #correct_answer = ord(correct_answer_letter)-ord('A')
+                        answers_dict = {}
```

Comment:
Cały ten for można by zamienić przez *dict comprehension* (https://www.python.org/dev/peps/pep-0274/).
```
answers_dict = {
    unichr(i+ord('A')): unicode(answer, "utf8")
    for i, answer in enumerate(row)
}
```